docs/reference/scripting/security.asciidoc

```diff
@@ -78,10 +78,11 @@ security of the Elasticsearch deployment.
 [float]
 === Allowed script types setting
 
-By default all script types are allowed to be executed.  This can be modified using the
-setting `script.allowed_types`.  Only the types specified as part of the setting will be
-allowed to be executed.  To specify no types are allowed, set `script.allowed_types` to
-be `none`.
+Elasticsearch supports two script types: `inline` and `stored` (<<modules-scripting-using>>).
+By default both script types are allowed to be executed. This can be modified using the
+setting `script.allowed_types`. Only the types specified as part of the setting will be allowed
+to be executed. To specify no types are allowed, set `script.allowed_types` to be `none`. To
+allow both types again, set `script.allowed_types: null` or `script.allowed_types: inline, stored`.
 
```

Comment:
"Allowed to be executed" feels awkward. Maybe:

By default, both types of scripts are allowed. To limit the types of scripts that can run, set `script.allowed_types` to `inline`, `stored`, or `none`. For example, to run `stored` scripts but not `inline` scripts, specify:

----
script.allowed_types: stored
----

Setting `script.allowed_types: null` or `script.allowed_types: inline, stored`allows both types of scripts to run.

Reply:
thanks Deb! much better 😄